ddev/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * Add color output to tests in CI ([#15774](https://github.com/DataDog/integrations-core/pull/15774))
 
+***Added***:
+
+* Migrate `ddev dep` to `ddev` ([#15830](https://github.com/DataDog/integrations-core/pull/15830))
+
```

Comment:
Any ideas how you ended up with two `Added` sections?

Reply:
Probably came about when rebasing, and apparently the changelog validation doesn't catch that; I've fixed it manually for now.